public/samples/DataFeeds/SVR/broadcaster.go

```diff
@@ -1,6 +1,8 @@
 import "github.com/flashbots/mev-share-node/mevshare"
 
 // ... create and sign your transaction
+// Your transaction should send ETH to block.coinbase as the bid
```

Comment:
i don't think we should put this info into the go code itself as it has to be done in solidity.